src/store/filestore.rs

```diff
@@ -151,6 +157,16 @@ impl LocalFileStore {
         });
         Ok(paths)
     }
+    /// Get the directory where store is located.
```

Comment:
Oh it got reorderd. Okay maybe let see if we need this, because can't these just be public?